actuators/actuators.go

```diff
@@ -71,6 +72,22 @@ func kubeconfigToSecret(clusterName, namespace string) (*v1.Secret, error) {
 		return nil, errors.WithStack(err)
 	}
 
+	allNodes, err := nodes.List(fmt.Sprintf("label=%s=%s", constants.ClusterLabelKey, clusterName))
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	// This is necessary so the management cluster in a container can talk to another container.
+	// They share the same bridged network and the load balancer does respond on 6443 at its docker IP
+	// however, the *HOST* is listening on some random port (the one returned from the GetLoadBalancerHostAndPort).
+	lbip, _, err := actions.GetLoadBalancerHostAndPort(allNodes)
+	lines := bytes.Split(data, []byte("\n"))
+	for i, line := range lines {
+		if bytes.Contains(line, []byte("https://")) {
+			lines[i] = []byte(fmt.Sprintf("    server: https://%s:%d", lbip, 6443))
```

Comment:
Hi @chuckha,

Should the port always be assumed?

Reply:
yes, the port should always be assumed as 6443. The apiservers will always be listening on their own IP at port 6443 until we allow users to customize support. This will change when support for that feature is added.